src/App.tsx

```diff
@@ -1,61 +1,111 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { Todo, Todos } from './types/todo';
+import { TodoList } from './components/TodoList';
 
 export const App = () => {
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+  const [todos, setTodos] = useState<Todos>(todosFromServer);
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState('0');
+
+  const getMaxId = (allTodos: Todos) => {
+    const newTodoId = Math.max(...allTodos.map(todo => todo.id));
+
+    return newTodoId + 1;
+  };
+
+  const handeTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
There is a typo in the function name `handeTitleChange`. It should be `handleTitleChange` to match the naming convention and avoid confusion.